src/get.brs

```diff
@@ -19,28 +19,28 @@
 '  * '  => invalid
 '  *
 '  */
-Function rodash_get_(aa, path, default=invalid)
+Function rodash_get_(array, path, default=invalid)
 
-  if aa = invalid or type(aa) <> "roAssociativeArray" then return default
+if array = invalid or not (type(array) = "roAssociativeArray" or type(array) = "roArray") then return default
 
   segments = m.pathAsArray_(path)
   if segments = invalid then return default
 
   result = invalid
   while segments.count() > 0
     key = segments.shift()
-    if not aa.doesExist(key)
+    value = array[key]
+    if value = invalid
       exit while
     end if
-    value = aa.lookup(key)
     if segments.count() = 0
       result = value
       exit while
     end if
-    if value = invalid or type(value) <> "roAssociativeArray"
+    if value = invalid or not (type(value) = "roAssociativeArray" or type(value) = "roArray")
```

Comment:
The `= invalid` check looks like it can be removed from this line since it is checked on line 33.

Reply:
Sorry for not getting back to this! Updated! Thanks for this great lib!